modules/stochastic_tools/include/state_sim/StateProcessor.h

```diff
@@ -0,0 +1,39 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#ifndef STATEPROCESSOR_H
+#define STATEPROCESSOR_H
+
+#include <string>
+#include <vector>
+
+using namespace std;
+
+class StateProcessor
+{
+  private:
+    int _max_time_step;
+    std::vector<int> _ev_times;
```

Comment:
opt for `unsigned int` whenever it's appropriate. This is because container indices are unsigned and you often compare against them in loops.